src/dashboard/cd-dashboard-events.vue

```diff
@@ -89,7 +124,12 @@
     async created() {
       await this.loadUserDojos();
       await this.loadEvents();
-      await this.loadDojos();
+      if (!this.events.length) {
+        // Reset to default for the animation to continue running
+        this.events = [{}, {}];
+        this.loadOldEvents();
+      }
+      this.loadDojos();
```

Comment:
This should actually probably only be called if we find upcoming events.

Reply:
Nope, we need it as well for oldEvents as we need the Dojo age to know what message to display

Reply:
Fair point